forge-gui/res/cardsfolder/b/boot_nipper.txt

```diff
@@ -3,8 +3,6 @@ ManaCost:1 B
 Types:Creature Beast
 PT:2/1
 K:ETBReplacement:Other:CounterChoice
-SVar:CounterChoice:DB$ GenericChoice | Defined$ You | Choices$ Deathtouch,Lifelink | SpellDescription$ CARDNAME enters the battlefield with your choice of a deathtouch counter or a lifelink counter on it.
-SVar:Deathtouch:DB$ PutCounter | CounterType$ Deathtouch | CounterNum$ 1 | ETB$ True | SpellDescription$ Deathtouch
-SVar:Lifelink:DB$ PutCounter | CounterType$ Lifelink | CounterNum$ 1 | ETB$ True | SpellDescription$ Lifelink
-DeckHints:Ability$Counters
+SVar:CounterChoice:DB$ PutCounter | ETB$ True | CounterType$ Deathtouch,Lifelink | SpellDescription$ CARDNAME enters the battlefield your choice of a deathtouch counter or a lifelink counter on it.
```

Comment:
Missing "with"

Reply:
Ugh that might have been copy/pasted into many files